Assets/_Application/_Twin/UI/ColorPicker/ColorWheel.cs

```diff
@@ -250,5 +250,11 @@ private void PickColor()
         {
             
         }
+
+        public void ResetColor()
+        {
+            color = Color.white;
+            Color = color;
```

Comment:
Het 'unsetten' van de color zou een null value moeten zijn en niet wit - wit is namelijk een valide kleur om toe te kennen aan een object (vergelijk het met de color waarde none in CSS, of het witte cirkeltje met de rode streep in een tekenprogramma).

Deze null value zou als effect moeten hebben dat de fill-color of stroke-color (afhankelijk van het type object) geunset moet worden (op null gezet iirc) zodat er geen override meer aanwezig is